typegen/strats/inline.py

```diff
@@ -53,16 +73,100 @@ def _innermost_function(self) -> cst.FunctionDef | None:
         first: cst.FunctionDef | None = next(fdefs, None)  # type: ignore
         return first
 
-    def _find_targets(
+    def _get_trace_for_targets(
         self, node: cst.Assign | cst.AnnAssign | cst.AugAssign
-    ) -> list[tuple[str, cst.Name | cst.Attribute]]:
-        extractor = TargetExtractor()
-        if isinstance(node, cst.AnnAssign | cst.AugAssign):
-            node.target.visit(extractor)
+    ) -> tuple[pd.DataFrame, pd.DataFrame, Targets]:
+        """
+        Fetches trace data for the targets from the given assignment statement.
+        If the statement references members of classes, but is NOT within a
+        method of a class with these members, then we do not NOT return their data.
```

Comment:
This functionality should be directly in the tracer. Or rather, how does the tracer even successfully trace member assigns of a local variable? This should be a bug.